content/administration/maintain/domain_names.rst

```diff
@@ -0,0 +1,324 @@
+   - Your domain name is **free for one year** if you register it with Odoo!
+   - The domain name is registered with `Gandi <https://www.gandi.net/>`_, the domain name
+     registrar.
+   - You are the owner of the domain name and can use it for other purposes.
+   - Odoo manages payment and technical support for you.
+
+To do so, go to :menuselection:`Website --> Go to website --> Promote --> Domain Name`.
+Alternatively, open your `database manager <https://www.odoo.com/my/databases>`_, click on the
+*settings* button next to your database, then on *Domain names*.
+
+.. image:: domain_names/register-promote.png
+   :align: center
+   :alt: Clicking on Domain Names from an Odoo website
+
+Search for the domain name of your choice to check its availability, then select the one you want to
+register for your website.
+
+.. image:: domain_names/register-search.png
+   :align: center
+   :alt: The search of the domain name example.com shows which associated domains are available.
+
+Next, fill in the form with your information to become the domain name owner.
+
+Your domain name is directly linked to your database, but you still have to :ref:`map your domain
+name with your website <domain-name/website-map>`.
+
+.. note::
+   - Free domain names are also available for free Odoo Online databases (if you installed one app
+     only, for example). In this case, Odoo reviews your request and your website to avoid abuse.
+     This process may take up to three days.
+   - This is not available for Odoo.sh databases yet.
+
+.. _domain-name/odoo-manage:
+
+Manage your domain name registered with Odoo
+--------------------------------------------
+
+To manage the DNS records of your domain name registered with Odoo or to visualize the contacts
+associated with it, open your `database manager <https://www.odoo.com/my/databases>`_, click on the
+*settings* button next to your database, on *Domain names*, and then on *Contacts* or *DNS*.
+
+.. image:: domain_names/manage.png
+   :align: center
+   :alt: Management of the domain names linked to an Odoo database
+
+.. note::
+   Please `submit a support ticket <https://www.odoo.com/help>`_ if you need further assistance to
+   manage your domain name.
+
+.. _domain-name/existing:
+
+Configure your existing domain name
+===================================
+
+If you already own a domain name, you can use it to point to your website.
+
+To avoid any issue with the :ref:`SSL certificate validation <domain-name/ssl>`, we highly recommend
+that you proceed with the following actions in this order:
+
+#. :ref:`Add a CNAME record <domain-name/cname>` on your domain name's DNS zone.
+#. :ref:`Map your domain name with your Odoo database <domain-name/db-map>`.
+#. :ref:`Map your domain name with your Odoo website <domain-name/website-map>`
+
+.. _domain-name/cname:
+
+Add a CNAME record
+------------------
+
+A **CNAME record** is a type of DNS record that points to the IP address of another website address
```

Comment:
A **CNAME record** is a type of DNS record that points to the address of another website